crates/bevy_reflect/src/impls/std.rs

```diff
@@ -28,6 +28,113 @@ impl_reflect_value!(Option<T: Serialize + Clone + for<'de> Deserialize<'de> + Re
 impl_reflect_value!(HashSet<T: Serialize + Hash + Eq + Clone + for<'de> Deserialize<'de> + Send + Sync + 'static>(Serialize, Deserialize));
 impl_reflect_value!(Range<T: Serialize + Clone + for<'de> Deserialize<'de> + Send + Sync + 'static>(Serialize, Deserialize));
 
+macro_rules! impl_reflect_tuple {
```

Comment:
nit: this barely matters, but I think we should move this to `tuple.rs`. tuples don't rely on any std types. they are a core part of the language.